astropy/extern/plugins/pytest_doctestplus/output_checker.py

```diff
@@ -118,7 +118,17 @@ def normalize_floats(self, want, got, flags):
                 else:
                     nw_.append(nw)
 
-                if not np.allclose(float(ng), float(nw)):
+                ngf = float(ng)
+                nwf = float(nw)
+
+                # This is a workaround for the backported version of this
+                # plugin since we need to support older versions of numpy that
+                # do not provide the 'equal_nan' argument, which was added in
+                # numpy-1.10.
+                if np.isnan(ngf) and np.isnan(nwf):
```

Comment:
This looks OK, except just to be sure: since this is for astropy itself, we can just import `NUMPY_LT_1_10` from `astropy.utils.compat` somewhere above and do `if NUMPY_LT_1_10 and np.isnan(ngf) and np.isnan(nwf):`
-- that way we don't slow down the rest and also have an easier time removing this if it is no longer necessary.